applications/endoscopy_tool_tracking_distributed/cpp/README.md

```diff
@@ -21,13 +21,26 @@ If you want to manually convert the video data, please refer to the instructions
 
 ### Run Instructions
 
-Run the following command to start the application.  This will run the app with a pre-recorded video as input:
 
 ```sh
+# Build the Holohub container for the Distributed Endoscopy Tool Tracking application
+./dev_container build --docker_file applications/endoscopy_tool_tracking_distributed/Dockerfile
```

Comment:
Would recommend building/running with non-default tag to minimize confusion among environments:

```suggestion
./dev_container build --docker_file applications/endoscopy_tool_tracking_distributed/Dockerfile --img holohub:endoscopy_tool_tracking_distributed
```

And then launch with `--img holohub:endoscopy_tool_tracking_distributed` below